File: 75/singleNumber.java
public class singleNumber {
    public static void main(String[] args) {
        int[] nums = { 4, 1, 2, 1, 2 };
        System.out.println(singlenumber(nums));
    }

    public static int singlenumber(int[] nums) {
        int n = 0;
        for (int num : nums) {
            n = n ^ num;
        }
        return n;
    }
}

File: Topic Wise/Bit Manipulation/singleNumber.py
# Single Number LC 136
# Given a non-empty array of integers, every element appears twice except for one. Find that single one.

nums=[2,2,1]


def singleNumber(nums):
    # Using XOR operator
    a = 0
    for i in nums:
        a ^= i
    return a

print(singleNumber(nums))